entrypoint

```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+set -e
+
+RUN_COMMAND="talisker.gunicorn webapp.app:app --bind $1 --name talisker-`hostname`"
```

Comment:
@samhotep requested that we don't use Gunicorn this time and just go with talisker

Reply:
There seems to  some issue in the later stages for him to manage, you may contact him for more info.